app/experimenter/experiments/bugzilla.py

```diff
@@ -84,6 +92,15 @@ def create_experiment_bug(experiment):
     return response_data["id"]
 
 
+def get_firefox_major_version(version):
+    return version.split(".")[0]
+
+
+def get_bugzilla_id(experiment_url):
+    if experiment_url:
+        return int(experiment_url.split("id=")[1])
```

Comment:
Ah this is a little fragile, you can use `urllib.parse.urlparse` to pull out the query string and then `urllib.parse.parse_qs` to get the actual id field out:

https://docs.python.org/3/library/urllib.parse.html#urllib.parse.parse_qs